test/test-exec-deserialization.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/python3
+
+#
+#  Copyright 2017 Michal Sekletar <msekleta@redhat.com>
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+#
+#  systemd is distributed in the hope that it will be useful, but
+#  WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public License
+#  along with python-systemd; If not, see <http://www.gnu.org/licenses/>.
+
```

Comment:
Could you please add a comment explaining how to run the test? I have to run something like
```
make && make install && systemctl daemon-reexec && ./test-exec-deserialization
```